File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        List<Integer> ans=new ArrayList<>();
        int[] mpp=new int[nums.length];
        for(int i=0;i<nums.length;i++)
        {
            mpp[i]=0;
        }
        for(int i=0;i<nums.length;i++)
        {
            mpp[nums[i]-1]=1;
        }
        for(int i=0;i<nums.length;i++)
        {
            if(mpp[i]==0)
            ans.add(i+1);
        }
        return ans;

        
    }
}

File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<bool> ans(nums.size(),0);
        vector<int> res;
        for(int i=0;i<nums.size();i++)
        {
            ans[nums[i]-1]=1;
        }
        for(int i=0;i<ans.size();i++)
        {
            if(ans[i]==0)
            res.push_back(i+1);
        }
        return res;
        
    }
};